script.sublissimo/addon.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.sublissimo" name="Sublissimo" version="0.0.1" provider-name="AJVM">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+    </requires>
+    <extension point="xbmc.python.script" library="main.py">
+        <provides>video</provides>
```

Comment:
revisiting @basrieter he is partially right. This addon does not provide video content. Please change it to `executable`